lib/services/firewall.pm

```diff
@@ -30,13 +30,13 @@ sub install_service {
 }
 
 sub susefirewall2_to_firewalld {
-    my $timeout = 180;
+    my $timeout = 360;
     assert_script_run('susefirewall2-to-firewalld -c',                                     timeout => $timeout);
```

Comment:
Yes, add some comments here will make reviewer easy to understand. I suggest to set '360' here, while '60' set as $timeout and $timeout set to 60,  or just not use this variable.